process/block/postprocess/basePostProcess.go

```diff
@@ -159,7 +159,7 @@ func (bpp *basePostProcessor) GetCreatedInShardMiniBlock() *block.MiniBlock {
 		return nil
 	}
 
-	return bpp.intraShardMiniBlock.Clone()
+	return bpp.intraShardMiniBlock.DeepClone()
```

Comment:
not sure if this will add incompatibility issues

Reply:
will not add - I changed it at elrond-go-core. I just renamed the original Clone function to DeepClone.